pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -80,27 +82,31 @@ func ValidateHardPodAffinityWeight(path *field.Path, w int32) error {
 
 // ValidateNodeLabelArgs validates that NodeLabelArgs are correct.
 func ValidateNodeLabelArgs(args config.NodeLabelArgs) error {
-	if err := validateNoConflict(args.PresentLabels, args.AbsentLabels); err != nil {
-		return err
-	}
-	if err := validateNoConflict(args.PresentLabelsPreference, args.AbsentLabelsPreference); err != nil {
-		return err
-	}
-	return nil
+	var allErrs field.ErrorList
+
+	allErrs = append(allErrs, validateNoConflict(args.PresentLabels, args.AbsentLabels,
+		field.NewPath("presentLabels"), field.NewPath("absentLabels"))...)
+	allErrs = append(allErrs, validateNoConflict(args.PresentLabelsPreference, args.AbsentLabelsPreference,
+		field.NewPath("presentLabelsPreference"), field.NewPath("absentLabelsPreference"))...)
+
+	return allErrs.ToAggregate()
 }
 
 // validateNoConflict validates that presentLabels and absentLabels do not conflict.
-func validateNoConflict(presentLabels []string, absentLabels []string) error {
-	m := make(map[string]struct{}, len(presentLabels))
-	for _, l := range presentLabels {
-		m[l] = struct{}{}
+func validateNoConflict(presentLabels []string, absentLabels []string, presentPath *field.Path, absentPath *field.Path) field.ErrorList {
```

Comment:
This might be better renamed to something that indicates it's specifically for validating present/absent label conflicts, but I wouldn't block on that

Reply:
This function name is not created by me. But I think such function name is enough because if we want to present it's functionality (validating present/absent label conflicts), it may be too long. Comment above the function already says it.